src/main/java/frc/robot/RobotContainer.java

```diff
@@ -52,16 +64,22 @@ public RobotContainer()
                         new ModuleIOSparkMax(Constants.CAN.MODULE_BL_DRIVE, Constants.CAN.MODULE_BL_ROTATE, Constants.AIO.MODULE_BL_SENSOR, Constants.Drive.MODULE_BL_OFFSET),
                         new ModuleIOSparkMax(Constants.CAN.MODULE_BR_DRIVE, Constants.CAN.MODULE_BR_ROTATE, Constants.AIO.MODULE_BR_SENSOR, Constants.Drive.MODULE_BR_OFFSET)
                 );
+                _shooterBed = new ShooterBed(new ShooterBedIOSparkMax(9, 10, 4, Constants.ShooterBed.BED_ANGLE_OFFSET));
+                _shooterFlywheel = new ShooterFlywheel(new ShooterFlywheelIOSparkMax(11, 12)); // FIXME: Set correct IDs
```

Comment:
The constructor was updated, so this won't compile anymore.

```suggestion
                _shooterBed = new ShooterBed(new ShooterBedIOSparkMax(Constants.ShooterBed.BED_ANGLE_OFFSET));
                _shooterFlywheel = new ShooterFlywheel(new ShooterFlywheelIOSparkMax());
```